client/src/components/App.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import axios from 'axios';
+
+import styled from 'styled-components';
+
+import QuestionList from './QuestionList.jsx';
+
+class App extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			questions: [],
+		};
+		this.getQuestions = this.getQuestions.bind(this);
+	}
+
+	componentDidMount() {
+		this.getQuestions();
+	}
+
+	getQuestions() {
+		axios.get('/api/products/questions')
+			.then((data) => this.setState({ questions: data.data }))
+			.catch((err) => console.log(err));
+	}
+
+
+	render() {
+		const Wrapper = styled.section`
+      margin: auto;
+      width: 50%;
+      position: relative;
+      padding: 10px;
+      font-family: Stuart, Georgia, serif;
+  `;
+
+		const QuestionButton = styled.button`
+    background-color: #006699;
+    color: white;
+    float: right;
+    border: none;
+    padding: 10px;
+    border-radius: 2px;
+    font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
+    font-size: 16px;
+  `;
+
+		const Sort = styled.div`
+    color: black;
+    float: right;
+    margin-bottom: 20px;
+    font-family: "Roboto","Helvetica Neue","Helvetica","Arial",sans-serif;
+    font-size: 14px;
+`;
+
+		const LoadMore = styled.button`
+    background-color: light-grey;
+    color: black;
+    border: 1px solid;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    width: 20%;
+`;
```

Comment:
I'm pretty sure you do not want to declare these here. You'll want to move them outside of the class or function declaration. See _Define Styled Components outside of the render method_ from thier docs
https://styled-components.com/docs/basics#coming-from-css